src/backends/backend/python/python.cc

```diff
@@ -1028,3 +1019,5 @@ TRITONBACKEND_ModelInstanceFinalize(TRITONBACKEND_ModelInstance* instance)
 }
 
 }  // extern "C"
```

Comment:
Can `extern "C"` be used within a namespace?

Reply:
It seems like it is ok but the namespace boundaries are not respected. See below:

https://stackoverflow.com/questions/28996944/extern-c-linkage-inside-c-namespace

Reply:
It should be ok in this case. The extern "C" causes the TRITONBACKEND_* functions to not be mangled. But because they are in the namespace the functions and classes they call don't need to explicitly have the namespace prefix(es).

Reply:
Okay